### PR TITLE
chore(release): Release v0.9.12

### PR DESCRIPTION
**Pull request by dustmop**
AKA `raw_umber_miniature_schnauzer`

Patch release 0.9.12 features a number of fixes to various qri features, most aimed at improving general quality-of-life of the tool, and some others that lay the groundwork for future changes.

## HTTP API Changes
Changed the qri api so that the `/get` endpoint gets dataset heads and bodies. `/body` still exists but is now deprecated.

## P2P and Collaberation
A new way to resolve peers and references on the p2p network.
The start of access control added to our remote communication API.
Remotes serve a simple web ui.

## General polish
Fix ref resolution with divergent logbook user data.
Working directories allow case-insensitive filenames.
Improve sql support so that dataset names don't need an explicit table alias.
The `get` command can fetch datasets from cloud.

See the `CHANGELOG.md` for the full description of changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
+# [](https://github.com/qri-io/qri/compare/v0.9.11...v) (2020-09-10)
+
+Patch release 0.9.12 features a number of fixes to various qri features, most aimed at improving general quality-of-life of the tool, and some others that lay the groundwork for future changes.
+
+## HTTP API Changes
+Changed the qri api so that the `/get` endpoint gets dataset heads and bodies. `/body` still exists but is now deprecated.
+
+## P2P and Collaberation
```
Comment (ramfox) [inline CHANGELOG.md:8]: Collaberation => Collaboration

**Review by ramfox — APPROVED** [review]
LGTM!

**Review by b5 — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
+# [](https://github.com/qri-io/qri/compare/v0.9.11...v) (2020-09-10)
```
Comment (b5) [inline CHANGELOG.md:1]:
```suggestion
# [v0.9.12](https://github.com/qri-io/qri/compare/v0.9.11...v0.9.12) (2020-09-10)
```

**Review by b5 — COMMENTED** [review]
lol prose leaves a little to be desired, but I'm happy to get this out the door. We'll get our _massive_ marketing department on these release notes.

.... cc @rgardaphe

**Pull request merged**